kitsune/sumo/management/commands/extract.py

```diff
@@ -0,0 +1,63 @@
+from django.core.management.base import BaseCommand
+from babel.messages.frontend import CommandLineInterface
+
+
+class Command(BaseCommand):
+    """
+    Allows you to extract strings for localization.
+    Extract creates two .pot files - django.pot and djangojs.pot - that contain
+    messages to merge into the existing l10n files.
+
+    """
+
+    help = "Extracts localizable strings from the codebase."
+
+    def handle(self, *args, **options):
+        CommandLineInterface().run(
+            [
+                "pybabel",
+                "extract",
+                "-F",
+                "babel.cfg",
+                "-o",
+                "locale/templates/LC_MESSAGES/django.pot",
+                "-k",
+                "_lazy",
+                "-k",
+                "pgettext_lazy",
+                "-c",
+                "L10n",
```

Comment:
I also discovered that unless we do the following, a bunch of comments that start with `L10n:` are ignored.
```suggestion
                "L10n,L10n:",
```

Reply:
Awesome